### PR TITLE
Improve godoc documentation

### LINKED ISSUE #8: Improve documentation
- One section on the `logger` package
- One section on the `prompter` package
- One section on the `symbols` package

Describe the feature in a cleaner way than the current list of features.

### LINKED ISSUE #13: Fix godoc
Add a `disgo.go` file at the root of the repository for documentation purposes.

Currently, the godoc link is broken because there is no package in the root of the repo.

### PR DESCRIPTION
**Pull request by Ullaakut**
## Goal of this PR

This PR adds a bunch of comments to packages and attributes in order to improve documentation overall, and it also adds a `disgo` package at the root of the repository for documentation purposes (in a similar way to what [testify](https://godoc.org/github.com/stretchr/testify) does for example)

Fixes #13 
Closes #8 

## How to test it

- Ensure that I didn't make too many spelling mistakes
- Feel free to improve sentences / suggest better documentation / more precise vocabulary

### CODE REVIEW

File: prompter/prompter.go [hunk 1]
```diff
@@ -7,9 +9,18 @@ import (
...
 type Prompter struct {
+	// Writer on which the prompt's label and choices are
+	// written when prompting them.
 	writer io.Writer
+
+	// Reader from which the user's response to the prompt is
+	// read.
 	reader *bufio.Reader
 
+	// Whether or not this prompter is running in a TTY. If this is
```
Comment (jlevesy) [inline prompter/prompter.go]: Not exactly, you might want to answer the default value anytime even if you are  in a TTY.

File: disgo.go [hunk 2]
```diff
@@ -0,0 +1,22 @@
+// Package disgo is a logging and prompting library for
+// modern command line interfaces.
+//
+// It is NOT a substitute for a structured logger
+// in an application or a microservice, since it is
```
Comment (jlevesy) [inline disgo.go]: Heh, a microservice is an application :p

File: disgo.go [hunk 3]
```diff
@@ -0,0 +1,22 @@
+// Package disgo is a logging and prompting library for
+// modern command line interfaces.
+//
+// It is NOT a substitute for a structured logger
```
Comment (jlevesy) [inline disgo.go]: It is NOT intended to be an application logger.

File: disgo.go [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+// in an application or a microservice, since it is
+// completely centered around user interaction and
```
Comment (jlevesy) [inline disgo.go]: It does not provide structured logging and is not built with performance in mind.

**Review by jlevesy — DISMISSED** [review]
Some minor feedbacks  and suggestions. 

Otherwise, LGTM !

File: prompter/prompter.go (hunk 1 above)
Comment (Ullaakut) [reply prompter/prompter.go]: Mh 🤔What comment would you suggest then?

File: disgo.go (hunk 2 above)
Comment (Ullaakut) [reply disgo.go]: That's right, will edit this out :p

File: disgo.go (hunk 4 above)
Comment (Ullaakut) [reply disgo.go]: Ah yes, performance is also an important thing to mention. Thanks for the feedback 🙏🏽

**Pull request merged**